sdks/python/test-suites/gradle.properties

```diff
@@ -23,34 +23,34 @@
 # dataflow test-suites
 # (TODO): https://github.com/apache/beam/issues/21971
 # Add python 3.10 to dataflow test-suites
-dataflow_precommit_it_task_py_versions=3.9,3.12
+dataflow_precommit_it_task_py_versions=3.9,3.13
```

Comment:
Could be a follow up (non-blocking here) - it would be nice to refactor 3.9 and 3.13 into `max_py_version` and `min_py_version` variables